index.js

```diff
@@ -0,0 +1,185 @@
+const allTripsURL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+//______________________________________________________
+// STATUS MANAGEMENT:
+const reportStatus = (message) => {
+  $('#status-message').addClass('callout primary');
+  $('#status-message').html(message);
```

Comment:
Just a note, for me the browser jumps down below this element so I didn't see the error message until I browsed up.